### ISSUE TITLE
new Phalcon\Config\Adapter\Ini raises a warning

### ISSUE THREAD

**Issue by JarbasTomaz** [phalcon/cphalcon#10500]
```php
$ini_file = "path\\to\\config.ini";
echo "<pre>Config file: $ini_file</pre>\r\n";
$config = new \Phalcon\Config\Adapter\Ini($ini_file);
die("<pre>End of test.</pre>\r\n");
```

With XDebub reports:
"Warning: array_merge_recursive() expects at least 1 parameter, 0 given in ..."

**Comment by andresgutierrez** [comment]
Could you please post the contents of the config.ini file? Are you using Phalcon 2.0.3?

**Comment by JarbasTomaz** [comment]
Phalcon Version: 2.0.2

Config.ini:

[app]

[modulos]
home 		= /
ccq			= /
rh			= /

[ambiente]
desenvolvimento = 1

[controle_acesso]
timeout_em_segundos = 300
minimo_caracteres_senha = 8
 
[bibliotecas]
diretorio = xxxxxx

**Comment by andresgutierrez** [comment]
This is already fixed in 2.0.3

**Comment by JarbasTomaz** [comment]
OK. Good to know. The functionallity wasn't affected, anyway.
